src/common/prop.c

```diff
@@ -573,6 +573,55 @@ daos_prop_entry_get(daos_prop_t *prop, uint32_t type)
 	return NULL;
 }
 
+int
+daos_prop_entry_set_str(daos_prop_t *prop, uint32_t type, const char *str, daos_size_t len)
```

Comment:
A suggestion: It might be better to refactor the guts of this out into another function that handles setting the string on a `struct daos_prop_entry` pointer, and call that one `daos_prop_entry_set_str()`. This one could be named `daos_prop_set_str()`.

I think doing it that way would allow us to get rid of the helpers in https://github.com/daos-stack/daos/blob/master/src/control/cmd/daos/property.h#L59

Reply:
i see. added.